web-registry/src/lib/ecocredit/api.ts

```diff
@@ -240,7 +240,7 @@ export const getBridgedEcocreditsForAccount = async (
         const txStatus = await getBridgeTxStatus(res.txResponses[i].txhash);
 
         status = txStatus?.status;
-        destinationTxHash = txStatus?.destination_tx_hash;
+        destinationTxHash = txStatus?.emv_tx_hash;
```

Comment:
what about also renaming the variable ?
```suggestion
        evmTxHash = txStatus?.evm_tx_hash;
```

Reply:
I thought it was fine here to keep `destination` here as on the front end we do not necessarily need to know that it is an EVM tx hash.

Reply:
ok for me too but let's fix the typo